ci/Dockerfile.lambda

```diff
@@ -22,11 +22,21 @@ RUN python -m venv ${POETRY_HOME} \
 
 COPY pyproject.toml poetry.lock ${TASK_ROOT}/
 
+# Install Poetry dependencies first (including setuptools and wheel)
 RUN python -m venv ${APP_VENV} \
     && . ${APP_VENV}/bin/activate \
     && poetry install \
     && poetry add awslambdaric wheel
 
+# Now install NewRelic manually using the setuptools from Poetry
+RUN . ${APP_VENV}/bin/activate \
+    && wget https://download.newrelic.com/python_agent/release/newrelic-11.0.1.tar.gz \
+    && tar -xzf newrelic-11.0.1.tar.gz \
+    && cd newrelic-11.0.1 \
+    && python setup.py install \
+    && cd .. \
+    && rm -rf newrelic-11.0.1 newrelic-11.0.1.tar.gz
```

Comment:
Yeah so the previous approach was to isolate this with the master switch too. We can try not to, but we need to test **thoroughly** he master switch and see if the lambdas are getting stuck to their 1 minute timeout. It's possible that during the incident, we didn't remove an import (and just imports would be enough to trigger the timeout).